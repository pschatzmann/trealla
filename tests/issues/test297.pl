main :-
	A=A-A,
	term_variables(A, L),
	write(L), nl.

:- initialization(main).
