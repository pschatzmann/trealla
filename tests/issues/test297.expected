[_7,_8,_9]
